aiinfra-cluster/modules/aiinfra-compute/main.tf

```diff
@@ -208,3 +210,26 @@ resource "google_compute_instance_group_manager" "mig" {
     update = "30m"
   }
 }
+
+module "aiinfra-gke" {
+  source                   = "../gke-cluster"
+  count                    = var.enable_gke ? 1 : 0
+  project                  = var.project_id
+  region                   = var.region
+  zone                     = var.zone
+  name                     = "${local.resource_prefix}-gke"
+  network_self_link        = var.network_self_link
+  subnetwork_self_link     = var.subnetwork_self_link
+  node_service_account     = lookup(var.service_account, "email", null)
+  node_pools               = [
+    {
+      name                    = "system-nodes"
+      nodes_initial           = var.instance_count
+      nodes_min               = 1
```

Comment:
We need to make the following configurable:

1. Number of NodePools
2. Min number of Nodes in each
3. Desired number of Nodes in each
4. Base name of the NodePools